src/main/java/org/apache/cassandra/distributed/shared/AbstractBuilder.java

```diff
@@ -54,6 +54,7 @@
     private ClassLoader sharedClassLoader = Thread.currentThread().getContextClassLoader();
     private int broadcastPort = 7012;
     private BiConsumer<ClassLoader, Integer> instanceInitializer = (cl, id) -> {};
+    private int datadirCount = 3;
```

Comment:
not sure if we should stay at 1 (match previous logic) or keep this... would make sure tests run with multi dir by default

Reply:
yeah I want it to be 3 by default to make sure we exercise the range splitting code